docs/developer-docs/latest/development/admin-customization.md

```diff
@@ -227,7 +227,43 @@ Note also that the main navigation logo uploaded via the admin panel supersedes
 
 #### Favicon
 
-To update the favicon, put a favicon file in the `./src/admin/extensions` folder and update the `config.head.favicon` key in the [admin panel configuration](#configuration-options).
+Replacing the Strapi favicon with a custom favicon.
+:::tip
+This same process may be used to replace the login logo `AuthLogo` and menu logo with `MenuLogo`. For more details, please visit Strapi [#logos](#)
+:::
+
+1. Create an extensions folder at:
+`src/admin/extensions/`
```

Comment:
```suggestion
```

This line can be deleted as it's best to combine it with the previous one (same for all the examples below)